packages/geolocation/src/utils/welcomePackage.js

```diff
@@ -0,0 +1,5 @@
+const geolocationWelcome = () => {
+    console.log("Welcome to geolocation squad 😂🧭")
+}
+
+module.exports = geolocationWelcome
```

Comment:
@Edilberto-Vazquez Can we add the EOL here?

Reply:
I already add the EOL